packages/web/src/common/store/tipping/slice.ts

```diff
@@ -22,7 +22,10 @@ const initialState: TippingState = {
     user: null,
     amount: new BN('0') as BNWei,
     error: null
-  }
+  },
+  recentTips: [],
+  recentTip: null,
```

Comment:
Wasn't clear what the difference between these two fields was based on the names - maybe something like `recentTips` and `tipToDisplay` would help?